Xamarin.Essentials/Geolocation/Geolocation.tizen.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Tizen.Location;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Geolocation
+    {
+        static Location lastKnownLocation = new Location();
+
+        static async Task<Location> PlatformLastKnownLocationAsync()
+        {
+            await Permissions.RequestAsync(PermissionType.LocationWhenInUse);
+
+            return lastKnownLocation;
+        }
+
+        static async Task<Location> PlatformLocationAsync(GeolocationRequest request, CancellationToken cancellationToken)
+        {
+            await Permissions.RequireAsync(PermissionType.LocationWhenInUse);
+
+            Locator service = null;
+            var gps = Platform.GetFeatureInfo<bool>("location.gps");
+            var wps = Platform.GetFeatureInfo<bool>("location.wps");
+            if (gps)
+            {
+                if (wps)
+                    service = new Locator(LocationType.Hybrid);
+                else
+                    service = new Locator(LocationType.Gps);
+            }
+            else
+            {
+                if (wps)
+                    service = new Locator(LocationType.Wps);
+                else
+                    service = new Locator(LocationType.Passive);
+            }
+
+            var tcs = new TaskCompletionSource<bool>();
+
+            cancellationToken = Utils.TimeoutToken(cancellationToken, request.Timeout);
+            cancellationToken.Register(() =>
+            {
+                service?.Stop();
+                tcs.TrySetResult(false);
+            });
+
+            service.LocationChanged += (s, e) =>
+            {
+                if (e.Location != null)
```

Comment:
Just checking the following:
 - [ ] Accuracy in meters?
 - [ ] Altitude in meters?
 - [ ] Speed in meters per second?

Reply:
- Accuracy : meters
- Altitude : meters
- Speed : km/h

https://samsung.github.io/TizenFX/API4/api/Tizen.Location.Location.html

Reply:
Can you please translate the value to meters / second to match the other platforms?

Reply:
I think this is the correct calculation to use, to set your `Location.Speed`

```csharp
double KmToMetersPerSecond(double km)
    => km * 0.277778;
```

Reply:
You are right. We will check the actual values once more (to verify whether the document is in error or not) and update them immediately.